massa-signature/src/signature_impl.rs

```diff
@@ -329,7 +329,7 @@ pub struct PublicKey(ed25519_dalek::PublicKey);
 
 const PUBLIC_PREFIX: char = 'P';
 
-#[allow(clippy::derive_hash_xor_eq)]
+#[allow(clippy::derived_hash_with_manual_eq)]
```

Comment:
what's this ?

Reply:
Clippy changed the lint name (see https://github.com/rust-lang/rust-clippy/pull/10184) to be more explicit.
This lint is allowed, otherwise we have to manually implement the Hash trait to avoid clippy warnings, IIRC.

Reply:
ok